automation/ansible/roles/setup_broker/templates/artemis-users.properties.j2

```diff
@@ -0,0 +1,27 @@
+{% if enable_ssl is defined and enable_ssl == true %}
+    r1-broker01-live=CN=r1-broker01-live,O=RedHat,C=CA
```

Comment:
You can use a for loop instead, e.g.

```
    {%- for brk in vars['play_hosts'] %}
        {{ brk }}=CN={{ brk }},O=RedHat,C=CA
    {% endfor %}
```

It'll use the hosts defined in the playbook